dm/dm-overview.md

```diff
@@ -71,7 +71,7 @@ You are welcome to participate in the DM open sourcing project. Your contributio
 You can learn about DM through the online documentation. If you have any questions, contact us in one of the following ways.
 
 - [GitHub](https://github.com/pingcap/tiflow/tree/master/dm)
-- [Slack](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=pingcap-docs)
+- [TiDB Internals forum](https://internals.tidb.io/)
```

Comment:
@luzizhuo @zhangyangyuMaybe we should remove this link as well because the title is *Community support* and the link is to help resolve users' questions, not for them to contribute.

Reply:
+1